README.md

```diff
@@ -54,7 +54,7 @@ or if you use the Kotlin DSL...
 
 ```kotlin
 // Kotlin DSL
-test {
```

Comment:
This can't be accessed in Kotlin DSL, we must use `tasks` as the extension receiver.